source/1_engine/rex_engine/src/diagnostics/logging/internal/common.cpp

```diff
@@ -6,74 +6,68 @@
 
 namespace rexlog
 {
-    namespace level
-    {
+  namespace level
+  {
 
-        static string_view_t g_level_string_views[]
-        {
-            REXLOG_LEVEL_NAME_TRACE, REXLOG_LEVEL_NAME_DEBUG, REXLOG_LEVEL_NAME_INFO, REXLOG_LEVEL_NAME_WARNING, REXLOG_LEVEL_NAME_ERROR, REXLOG_LEVEL_NAME_CRITICAL, REXLOG_LEVEL_NAME_OFF                                                                  \
-        };
+    const rsl::array<rsl::string_view, 7> g_level_string_views = {                          // NOLINT(fuchsia-statically-constructed-objects, cppcoreguidelines-avoid-non-const-global-variables, modernize-avoid-c-arrays)
```

Comment:
are all these clang-tidy nolints needed here?
modernize-avoid-c-arrays comes to mind that is not needed here, but I also think the others aren't needed
we need to check which clang-tidy checks we're going to globally disable btw